staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -259,9 +265,9 @@ func (c *AvailableConditionController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
-	// only start one worker thread since its a slow moving API and the aggregation server adding bits
```

Comment:
I think this comment was copy/pasted from the controller that tickles the proxy handler.  I didn't see any links out from here that looked dangerous.